File: spec/code_samples/Javascript jquery/keys/post.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/keys",
  "method": "POST",
  "headers": {
    "content-type": "application/json"
  },
  "processData": false,
  "data": "{\"exchange\":\"kraken\",\"key\":\"string\",\"secret\":\"string\",\"passphrase\":\"string\",\"userID\":\"string\"}"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: spec/code_samples/Python requests/keys/post.py
import requests

url = "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/keys"

payload = "{\"exchange\":\"kraken\",\"key\":\"string\",\"secret\":\"string\",\"passphrase\":\"string\",\"userID\":\"string\"}"
headers = {'content-type': 'application/json'}

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)